phaseshifts/model.py

```
@@ -594,16 +591,14 @@ def set_nform(self, nform):
                     self.nform = 2
                 elif nform in ["0", "1", "2"]:
                     self.nform = int(nform)
-        except any as e:
-            raise TypeError(e.msg)
+        except Exception as exc:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)